test/libraries.bats

```diff
@@ -99,6 +99,7 @@ function setup() {
   if [[ "$(crystal env CRYSTAL_VERSION)" =~ ^0\.|^1\.[0-8]\. ]]; then
     skiponwindows "Compiler bug in Crystal < 1.9"
   fi
+  skiponwindows "Specs are failing"
```

Comment:
What's the point of having the three lines above, if it's going to be skipped anyway?

Reply:
It's different reasons for skipping. This documents that it's not expected to build with Crystal 1.8 and below. But after that it's (probably) up to the shard wether it works or not.